scripts/qesap/test/unit/test_qesap_terraform.py

```diff
@@ -55,7 +59,7 @@ def test_terraform_terraform_logs(run, terraform_cmd_args, args_helper, config_y
     provider = 'mangiafuoco'
     conf = config_yaml_sample(provider)
 
-    args, terraform_dir, _, _ = args_helper(provider, conf, '')
+    args, *_ = args_helper(provider, conf, '')
```

Comment:
what is `*_` ?

Reply:
`*foobar` catches the rest of the returned values in an array.  `_` is just the throaway variable name by convention.